src/AutoMapper/Bundle/Configuration/ConfigurationPassInterface.php

```diff
@@ -4,6 +4,9 @@
 
 use Jane\AutoMapper\MapperGeneratorMetadataInterface;
 
+/**
+ * @deprecated
```

Comment:
Please add a message with what replaced this interface and on which version this interface will be dropped (v7.x I suppose).